addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -329,6 +336,17 @@ def action_get_attachment_tree_view(self):
         action['search_view_id'] = (self.env.ref('hr_recruitment.ir_attachment_view_search_inherit_hr_recruitment').id, )
         return action
 
+    @api.multi
+    def action_applications_email(self):
+        return {
+            'type': 'ir.actions.act_window',
+            'name': _('Applications fgdfsgf'),
```

Comment:
Applications fgdfsgf ?

Reply:
Don't you know 'Applications fgdfsgf' :p ? 
I'll correct this 
```suggestion
            'name': _('Applications'),
```